rm/rm-server/src/main/java/org/ow2/proactive/resourcemanager/nodesource/infrastructure/InfrastructureManager.java

```diff
@@ -531,6 +532,24 @@ public final void internalShutDown() {
      */
     protected abstract void configure(Object... parameters);
 
+    /**
+     * Reconfigure the infrastructure of a potentially already deployed node
+     * source with the given parameters. By default, this method overrides the
+     * infrastructure parameters regardless whether they are currently being
```

Comment:
regardless of their status / whether they are currently
you cannot use both

Reply:
Ok, done.